environment.yml

```
@@ -12,7 +12,7 @@ dependencies:
   - jupyter-book=0.15.1
   - pooch=1.7.0
   - scikit-learn=1.2.2
-
+  - ipywidgets==8.0.7
```

```suggestion
  - ipywidgets=8.0.7
  - seaborn=0.12.2
```
Used in the [segmentation tutorial](https://github.com/scipy-2023-image-analysis/tutorial/pull/10/files#diff-0ad01c72c8cfde87a9254911db2e1c2a39c22cc9ec5b6c534c4467c3f3750ac8R498).